packs/data/feats.db/imbue-mindlessness.json

```diff
@@ -0,0 +1,45 @@
+{
+    "_id": "uvbPZR6dsQBimIUo",
+    "data": {
+        "actionCategory": {
+            "value": ""
+        },
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "description": {
+            "value": "<p>You grasp the mind of an undead creature and temporarily wipe it as empty as a zombie’s brain. You can target any undead with your divine innate <em>@Compendium[pf2e.spells-srd.Harm]{Harm}</em> spell, not just mindless undead. When you target one non-mindless undead creature with this spell, you can transform its effects. Instead of harm’s normal effects, the target must attempt a Will save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 1} for 1 round.</p>\n<p><strong>Failure</strong> The target is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 2} for 1 minute. While it is stupefied, you can use your Corpse Tender feats on the target as though it were mindless.</p>\n<p><strong>Critical Failure</strong> As failure, except the target is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 3} for 10 minutes.</p>"
```

Comment:
```suggestion
            "value": "<p>You grasp the mind of an undead creature and temporarily wipe it as empty as a zombie's brain. You can target any undead with your divine innate <em>@Compendium[pf2e.spells-srd.Harm]{Harm}</em> spell, not just mindless undead. When you target one non-mindless undead creature with this spell, you can transform its effects. Instead of harm's normal effects, the target must attempt a Will save.</p>\n<hr />\n<p><strong>Critical Success</strong> The target is unaffected.</p>\n<p><strong>Success</strong> The target is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 1} for 1 round.</p>\n<p><strong>Failure</strong> The target is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 2} for 1 minute. While it is stupefied, you can use your Corpse Tender feats on the target as though it were mindless.</p>\n<p><strong>Critical Failure</strong> As failure, except the target is @Compendium[pf2e.conditionitems.Stupefied]{Stupefied 3} for 10 minutes.</p>"
```